datastore/bolt/repo_test.go

```diff
@@ -1,24 +1,132 @@
 package bolt
 
 import (
+	"github.com/drone/drone/common"
+	"github.com/drone/drone/common/sshutil"
 	"testing"
-
-	. "github.com/franela/goblin"
+	//. "github.com/franela/goblin"
+	. "github.com/smartystreets/goconvey/convey"
 )
 
 func TestRepo(t *testing.T) {
-	g := Goblin(t)
+	/*g := Goblin(t)
 	g.Describe("Repos", func() {
 
-		g.It("Should find by name")
+		g.It("Should find by name") // ok
 		g.It("Should find params")
 		g.It("Should find keys")
-		g.It("Should delete")
-		g.It("Should insert")
-		g.It("Should not insert if exists")
+		g.It("Should delete") // ok
+		g.It("Should insert") // ok
+		g.It("Should not insert if exists")  // ?
 		g.It("Should insert params")
 		g.It("Should update params")
 		g.It("Should insert keys")
 		g.It("Should update keys")
 	})
+	*/
+	var testUser string = "octocat"
```

Comment:
no need to declare types here. Go infers them automatically. Using `var` is also unnecessary in this case:
``` 
testUser := "octocat"
```